### DISCUSSION TITLE
Do I need to follow this procedure to get bluetooth to work?

### DISCUSSION THREAD

**Discussion by stevenperich** [timeylies/How-to-convert-a-Ninebot-SNSC-2.2-or-2.3-Scooter#3] [Q&A] [answered]
Hi there, I'm wondering if I need to follow this process for my scooter. I legally acquired an old SNSC 2.2 and the company I bought it from told me that the IoT board has already been removed and it never had bluetooth so I wont be able to enable it.  From what I can ascertain your process dumps BLE555 on the dashboard and with BLE in the name I figured it could be a way to enable the scooter to be managed like a G30 Max Ninebot in the normal app.  This is so I can ideally see the odometer details and whatever else....but also with Bluetooth enabled it seems I can run xiaoflasher and override the 25km/h limit that my scooter has..  The scooter works fine otherwise without any 'smarts'. 

Is any of this possible to do with the procedure outlined on this Wiki?   Or am I missing some vital piece of hardware like the BT radio?

**Comment by timeylies** [comment, marked as answer]
The BLE on the IOT is just for configuration, you don’t need it, all you need is the dash to have a Bluetooth radio, since the dash is named BLE after all

**Reply by stevenperich** [reply]
cool, thanks.  So effectively it does still have BLE on the dash, it's just that the firmware it shipped with never had it enabled, and reflashing with BLE555 will enable that?

**Reply by timeylies** [reply]
yeah

**Reply by stevenperich** [reply]
Hi again. Sooooo I downloaded and had a look at BLE555 firmware and all the zip files available (eg from the link you have in your wiki) don't have the .bat file in them to make it work properly. As above my scooter had the dashboard present when I bought it so as I understand I could reflash (not easy without the missing .bat file!) or purchase a new dash.  But if I do that will I have to rewire the whole thing?

**Reply by timeylies** [reply]
Nope, you wouldn’t have to rewire anything

**Reply by stevenperich** [reply]
Okay., thanks heaps. Just trying to avoid bricking my scooter!  So this would be the correct process for me right (in this order)?

- flash ESC with esc126 using st-link
- flash BMS with bms134 using STVP
- replace dashboard with new one from Ali express?

no need to buy Julet connectors, rewire, etc?

Is this method going to work any better or worse than the 14-step process in the description in this video https://youtu.be/jnxeM6gvq_U?si=FC6vHlH-1WNE0fZZ ? This process uses some software called Ninebot-Flasher and custom firmware from scooterhacking …

**Reply by timeylies** [reply]
I doubt that video tutorial works for these 2.2s, and replacing the dash isn't needed if yours comes with one (not the iot dash combo, an actual ninebot max dashboard with the iot on the stem), all you would have to do is just flash that dashboard with BLE555. If the dash connector is a 5 pin, yes you would need to rewire, but if its a 4 pin, then no, you dont have to rewire, it's just plug and play.

**Reply by stevenperich** [reply]
Thanks! The “updated” process in the description of the video is “updated” / different to what’s in the video (where he uses BLE555). And yeah I just noticed the video was for SNSC 2.0 not 2.2.. hummmm… 

also noticed somewhere on https://www.scooterhacking.org/forum/viewtopic.php?t=266 I think someone said BLE555 wasn’t the best and suggested I think v1.1.0 (can’t find the comment now damn). Is that better to use if I keep my existing dash? If not, do you have the original BLE555 with the convert_dashboard.zip file? Really can’t find it anywhere. 

really appreciate the help!!

**Reply by timeylies** [reply]
It's your preference on which one you want to use, i'd go with BLE110, only reason why i say flash the dash with 555, is because of the batch file, but you could always downgrade the dash with Scooterhacking Utility, and if you have a existing ninebot max dash, flash it and use it.

**Reply by stevenperich** [reply]
Yeah forget about ninebot-flasher it ain’t gonna work. Can’t even see the scooter in BT devices lol. So you have the BLE555 with the batch file or nah?

So I only have the dash that came with the SNSC2.2… you saying I could get a normal dash from Ali to flash and use? Or use the one that’s already in the scooter? Lol. 

Also not sure where to find BLE110 :( any ideas?

**Reply by timeylies** [reply]
heres the link: https://cloud.cfw.sh/index.php/s/xLPGxQTk3D2ZbNx
and yeah you could, but if it already comes with one, you just need to flash it.
BLE110 is in here: https://firmware.scooterhacking.org/max/BLE/
